src/components/entity/state-badge.ts

```diff
@@ -79,6 +79,9 @@ export class StateBadge extends LitElement {
       </div>`;
     }
 
+    const cls = this.getClass();
+    if (cls) this.classList.add(cls);
```

Comment:
This only does `add`. Would the class be removed on the next render if it is not added again or does it stay there forever? We might need to do a `remove` of all the potential classes first